roles/cloud-bootstrap/tasks/create-users-projects.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Create bonnyci project
+  os_project:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci
+
+- name: Create bonnyci-admin group
+  os_group:
+    cloud: "{{ cloud }}"
+    name: bonnyci-admin
+
+- name: Add all users to bonnyci group
+  os_user_group:
+    cloud: "{{ cloud }}"
+    group: bonnyci
+    user: "{{ item.username }}"
+  with_items: users
+
+- name: Add all bonnyci group members to bonnyci project
+  os_user_role:
+    cloud: "{{ cloud }}"
+    role: member
```

Comment:
@j2sol I think the `user-projects` role assumes a slightly different approach to our account provisioning, but I do agree that the code is extremely similar. Do you think we should remove that once this is merged and fully tested?